frontend/src/components/orders/OfferSummary.vue

```diff
@@ -0,0 +1,70 @@
+<script setup lang="ts">
+import Card from "primevue/card";
+import Dropdown from "primevue/dropdown";
+import { computed, ref, watch } from "vue";
+
+import { updateOffer } from "@/backendClient";
+import useNotifications from "@/compositions/useNotifications";
+import { ArticleKind, OfferStatus } from "@/global/types/appTypes";
+import type { Offer } from "@/global/types/entities";
+import { calculateBrutto, calculatePrice } from "@/helpers/utils";
+
+const props = defineProps<{
+  offer: Offer;
+}>();
+const notifications = useNotifications();
+
+const offersType = Object.values(OfferStatus);
+
+let offerStatusValue = ref<OfferStatus>(props.offer.status);
+const allItemsSum = computed(() => {
+  return calculatePrice(props.offer.items, props.offer.offered_at);
+});
+
+watch(offerStatusValue, async () => {
+  await updateOffer(props.offer.id, {
+    status: offerStatusValue.value,
+  });
+
+  notifications.showUpdateOfferStatusNotification();
+});
+</script>
+<template>
+  <section class="flex flex-col gap-2 sm:flex-row sm:gap-8 items-center">
+    <p><span class="font-bold">Datum:</span> {{ offer.offered_at }}</p>
+    <p><span class="font-bold">Gültigkeit:</span> {{ offer.offer_valid_until }}</p>
+    <p class="font-bold">Status:</p>
+
+    <Dropdown
+      v-model="offerStatusValue"
+      :options="offersType"
+      placeholder="Anrede"
+      class="w-full md:w-[14rem]"
+    />
+  </section>
+  <p class="font-bold">Preis:</p>
```

Comment:
I am wondering how we can make this section a bit more clear. The wording change could be a first step.

```suggestion
  <p class="font-bold">Angebotspreis:</p>
```